pom.xml

```diff
@@ -12,6 +12,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.1</version>
+        </dependency>
```

Comment:
I don't think this is needed anymore after https://github.com/binance-chain/java-sdk/pull/3/files.